src/content/store-and-form-setup.md

```diff
@@ -26,27 +38,46 @@ export interface IBioSummary {
 
 ---
 
+## Reduce, reuse...
+
+Plan any other forms you want to add to your form state
+
+```ts
+export interface IForm {
+  character: ICharacter; // state.form.character
+  equipment: IEquipment; // state.form.equipment
+}
+
+export interface IEquipment {
+  weaponName?: string;
+  weaponType: string;
+  armorType: string;
+}
+```
+
+---
+
 ## Setting up actions
+
 Action payloads include the path to the form in the store
 
 ```ts
 export const saveForm = (path, value) => ({
   type: 'SAVE_FORM',
   payload: {
-    path,
-    value
+    path, // ['character'] would be form.character
+    value,
   }
 });
 ```
 
 ---
 
 ## Create form reducer
-Including a path in the payload makes this reducer reusable for other forms
+Our form changes will be merged at the provided path
 
 ```ts
 import { lensPath, assocPath, merge } from 'ramda';
```

Comment:
@danielfigueiredo You mentioned we could tidy this example up and use only `path` from Ramda?